frontend/src/app/testQueue/TestCardForm/TestCardForm.tsx

```diff
@@ -0,0 +1,563 @@
+                aria-label="Test date"
+                min={formatDate(new Date("Jan 1, 2020"))}
+                max={formatDate(moment().toDate())}
+                value={formatDate(moment(state.dateTested).toDate())}
+                onChange={(e) => {
+                  if (e.target.value) {
+                    dispatch({
+                      type: TestFormActionCase.UPDATE_DATE_TESTED,
+                      payload: e.target.value,
+                    });
+                  }
+                }}
+                required={true}
+                validationStatus={showDateTestedError ? "error" : undefined}
+                errorMessage={showDateTestedError && dateTestedError}
+              ></TextInput>
+            </div>
+            <div className="grid-col-auto display-flex">
+              <TextInput
+                className="flex-align-self-end no-left-border"
+                id={`test-time-${testOrder.patient.internalId}`}
+                name="test-time"
+                type="time"
+                aria-label="Test time"
+                data-testid="test-time"
+                step="60"
+                value={moment(state.dateTested).format("HH:mm")}
+                onChange={(e) => {
+                  if (e.target.value) {
+                    dispatch({
+                      type: TestFormActionCase.UPDATE_TIME_TESTED,
+                      payload: e.target.value,
+                    });
+                  }
+                }}
+                validationStatus={showDateTestedError ? "error" : undefined}
+              ></TextInput>
+            </div>
+          </div>
+        )}
+        <div className="grid-row grid-gap">
+          <div className="grid-col-auto">
+            {useCurrentTime && (
+              <Label className={"margin-top-3"} htmlFor={"current-date-time"}>
+                Test date and time{" "}
+                <abbr
+                  title="required"
+                  className={"usa-hint usa-hint--required"}
+                >
+                  *
+                </abbr>
+              </Label>
+            )}
+            <Checkbox
+              id={`current-date-time-${testOrder.patient.internalId}`}
+              name={"current-date-time"}
+              label={"Current date and time"}
+              checked={useCurrentTime}
+              onChange={(e) => {
+                setUseCurrentTime(e.target.checked);
+                dispatch({
+                  type: TestFormActionCase.UPDATE_DATE_TESTED,
+                  payload: e.target.checked ? "" : moment().toISOString(),
+                });
+              }}
+            ></Checkbox>
+          </div>
+        </div>
+        <div
+          className="grid-row grid-gap margin-top-2"
+          data-testid="device-type-dropdown-container"
+        >
+          <div className="grid-col-auto">
+            <Dropdown
+              id={`test-device-${testOrder.patient.internalId}`}
+              options={deviceTypeOptions}
```

Comment:
just a heads up in the original queue card, G&E implemented a check to make sure we don't show RSV devices https://github.com/CDCgov/prime-simplereport/blob/ed7620b94596269419adc6cc8e1b7bcd4fe7a7b8/frontend/src/app/testQueue/QueueItem.tsx#L762 we would want that here as well